### PR TITLE
chore(tools/netlify): add more preview URLs 

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

File: tools/netlify.js [hunk 1]
```diff
@@ -17,6 +17,8 @@ const SPECS = {
     "https://w3c.github.io/payment-request/",
     "https://w3c.github.io/gamepad/",
     "https://w3c.github.io/hr-time/",
+    "https://w3c.github.io/web-share/",
+    "https://w3c.github.io/trace-context/",
```
Comment (sidvishnoi) [inline tools/netlify.js:20]: This uses data-include and markdown extensively.

Comment (marcoscaceres) [reply tools/netlify.js:20]: wow... never heard of this spec

Comment (sidvishnoi) [reply tools/netlify.js:20]: Any more URLs you would like?

Comment (marcoscaceres) [reply tools/netlify.js:20]: Lets get badging in there too.. will suggest

File: tools/netlify.js [hunk 2]
```diff
@@ -14,9 +14,11 @@ const PROFILES = ["w3c", "geonovum", "dini"];
 
 const SPECS = {
   W3C: [
-    "https://w3c.github.io/payment-request/",
     "https://w3c.github.io/gamepad/",
     "https://w3c.github.io/hr-time/",
+    "https://w3c.github.io/payment-request/",
+    "https://w3c.github.io/trace-context/",
+    "https://w3c.github.io/web-share/",
```
Comment (marcoscaceres) [inline tools/netlify.js:21]:
```suggestion
    "https://w3c.github.io/badging/",
```

**Pull request merged**